.circleci/config.yml

```diff
@@ -186,3 +220,11 @@ workflows:
           filters:
             branches:
               only: master
+      - release:
+          requires:
+            - build_and_release_docker_image
+          filters:
+            branches:
+              ignore: /.*/
```

Comment:
What exactly does this match?

Reply:
Does not match any commits to master, matches tags only. We use it for Node packages auto-release.

https://github.com/googleapis/nodejs-pubsub/blob/c400db220d5fcf7bbcbf55927879d2c275b00132/.circleci/config.yml#L52